docs/md/directory-usage.md

```diff
@@ -0,0 +1,160 @@
+# Using WFL Across a Directory
+
+WFL supports starting workflows from a single file each--depending on the
+pipeline you specify, other inputs will be extrapolated (see WFL's docs
+for the specific pipeline for more information).
+
+If you have a set of files uploaded to a GCS bucket and you'd like to start
+a workflow for each one, you can do that via shell scripting.
+
+!!! warning "Note"
+    You will likely run into performance issues with WFL if you try to start
+    hundreds or thousands of workflows in a single request to WFL. You'll need
+    to split up the workflows into multiple workloads, tips for that are
+    [here](#other-notes).
+
+Suppose we have a set of CRAMs in a folder in some bucket, and we'd like to
+submit them all to WFL for ExternalExomeReprocessing (perhaps associated with
+some project or ticket, maybe PO-1234). We'll write a short bash script that
+will handle this for us.
+
+> Make sure you're able to list the files yourself! You'll need permissions
+> and you may need to run `gcloud auth login`
+
+## Step 1: List Files
+
+We need a list of all the files you intend to process. This'll depend on the
+location of the files, so here we'll use an example of
+`gs://broad-gotc-dev-wfl-ptc-test-inputs/`. We can use 
+[wildcards](https://cloud.google.com/storage/docs/gsutil/addlhelp/WildcardNames)
+to list out the individual files we'd like. Make some scratch file like
+`script.sh` and store the list of CRAMs in a variable:
+
+```bash
+# In script.sh
+
+CRAMS=$(gsutil ls 'gs://broad-gotc-dev-wfl-ptc-test-inputs/**.cram')
+```
+
+## Step 2: Format Items
+
+First, we need to turn that string output into an actual list of file paths.
+We can use `jq` to `split` intolines and `select` ones that are paths:
+
+```bash
+CRAMS=$(gsutil ls 'gs://broad-gotc-dev-wfl-ptc-test-inputs/**.cram')
```

Comment:
You don't need to include the previous example(s) here. If you want, you can have one final section about  "putting it all together"